src/Platforms/CUDA/cuBLAS_missing_functions.hpp

```diff
@@ -47,7 +47,7 @@ using cuBLAS_MFs_handle = cudaStream_t;
  * @param incy increment for the elements of y. It cannot be zero.
  * @param batch_count batch size
  */
-cuBLAS_MFs_status gemv_batched(cuBLAS_MFs_handle& handle,
+cuBLAS_MFs_status gemv_batched(cuBLAS_MFs_handle handle,
```

Comment:
since we're changing these lines anyway can we remove the useless typedef that only serves to make this files types not match those in the implementation file?

Reply:
Will fix it.

Reply:
Done.